optimism/src/mips/keccak/mod.rs

```diff
@@ -0,0 +1,67 @@
+pub mod column;
+
+pub const ZKVM_KECCAK_COLS: usize = 1965 + 6;
+pub const RATE: usize = 1088;
+pub const RATE_IN_BYTES: usize = RATE / 8;
+pub const DIM: usize = 5;
+pub const QUARTERS: usize = 4;
+
+fn grid_20(x: usize, q: usize) -> usize {
+    q + QUARTERS * x
```

Comment:
Please either inline the constant here or change the name. Personally I think it would be clearer to have
```rust
fn matrix_2d_4(x: usize, y: usize) -> usize {
    y + 4 * x
}
```
and similarly
```rust
fn matrix_3d_4_5(x: usize, y: usize, z: usize) -> usize {
    z + 4 * (y + 5 * x)
}
```
for both of the below.